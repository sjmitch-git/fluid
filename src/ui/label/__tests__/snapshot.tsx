import { render } from '@testing-library/react'
import { Label } from '..'

import Data from '@/data/dogs.json'
const data = Data[0]

it('renders Label unchanged', () => {
	const { container } = render(
		<Label label='Label Value'>
			<input
				type='text'
				className='form-input w-full'
			/>
		</Label>
	)

	expect(container).toMatchSnapshot()
})
