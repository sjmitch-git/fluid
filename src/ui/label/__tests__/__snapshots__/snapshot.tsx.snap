// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Label unchanged 1`] = `
<div>
  <label
    class="label group relative cursor-pointer w-full font-semibold text-base flex flex-col items-start gap-2 text"
    data-testid="label"
  >
    <span
      class="inline-block  "
    >
      Label Value
       
    </span>
    <div
      class="w-full col-span-2"
    >
      <input
        class="form-input w-full"
        type="text"
      />
    </div>
  </label>
</div>
`;
